### PR TITLE
Add editClasses and editStyles props to CodeSnippet

### PR DESCRIPTION
**Pull request by katydecorah**
This PR adds `editClasses` and `editStyles` props to the CodeSnippet component. These props allow us to change the styles and classes for the Edit button so we may change their position on the page.

This PR will also fix the styling of the proptType comments so they will be parsed by our catalog site.

## How to test

1. `npm ci`
2. `npm start`
3. Visit /CodeSnippet and look at the last test case: CodeSnippet with edit buttons, maxHeight, editClasses, and editStyles. This uses test case shows how we can set the editClasses and editStyles in Help.

## QA checklist

- [x] No errors logged to console.
- [x] Accessibility score in [Chrome DevTools Audit/Lighthouse](https://developers.google.com/web/tools/lighthouse#devtools) is 100% with Lighthouse version: `6.4.0`.
- [x] Component is accessible at mobile-size viewport.
- [x] Component is accessible at tablet-size viewport.
- [x] Component is accessible at laptop-size viewport.
- [x] Component is accessible at big-monitor-size viewport.

Open the test cases app locally on:

- [ ] Chrome, no errors logged to console.
- [ ] Firefox, no errors logged to console.
- [ ] Safari, no errors logged to console.
- [ ] Edge, no errors logged to console.
- [ ] Mobile Safari, no errors logged to console.
- [ ] Android Chrome, no errors logged to console.

## Before merge

- [x] Add entry to CHANGELOG.md to describe changes.
- [ ] If updating dependencies or creating a release, run `npx browserslist@latest --update-db` to update the [browser data](https://github.com/browserslist/browserslist#browsers-data-updating) and commit any changes to package-lock.json.

### CODE REVIEW

File: src/components/code-snippet/__tests__/code-snippet-test-cases.js [hunk 1]
```diff
@@ -64,4 +64,26 @@ testCases.withHelpers = {
...
 };
 
+testCases.maxHeight = {
+  component: CodeSnippet,
+  description:
+    'CodeSnippet with edit buttons, maxHeight, editClasses, and editStyles.',
+  props: {
+    code: fullHtml,
+    maxHeight: 150,
+    editClasses: 'absolute-mm right mb3 mt-neg36-mm',
```
Comment (colleenmcginnis) [inline src/components/code-snippet/__tests__/code-snippet-test-cases.js:74]: So is the idea here that I would add `editClasses='absolute-mm right mb3 mt-neg36-mm'` to the `FullDemoCode` component (i.e. the buttons will not auto reposition if `maxHeight` is used)?

Comment (katydecorah) [reply src/components/code-snippet/__tests__/code-snippet-test-cases.js:74]: @colleenmcginnis ahh good point! I got the design backwards.Seems like when `maxHeight` is set, then we should automatically move the EditButtons (if enabled) above the CodeSnippet. I'll update.

Comment (katydecorah) [reply src/components/code-snippet/__tests__/code-snippet-test-cases.js]: @colleenmcginnis ok updated!

**Review by colleenmcginnis — APPROVED** [review]
Looks good to me! ⭐ 👩‍🍳

**Pull request merged**